manifests/params.pp

```diff
@@ -168,6 +168,7 @@
   $store_usage              = undef
   $temp_usage               = undef
   $disable_update_checking  = undef
+  $automatic_dlo_cleanup    = true
```

Comment:
We should default this to false and people can turn it on as needed for now. After it has been used a bit we can come back and look at defaulting to on.